app/javascript/dashboard/components/helpCenter/Sidebar/SidebarHeader.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="sidebar-header--wrap">
+    <div class="header-left--side">
+      <thumbnail
+        size="40px"
+        :src="thumbnailSrc"
+        :username="headerTitle"
+        variant="square"
+      />
+      <div class="header-title--wrap">
+        <h4 class="sub-block-title title-view">{{ headerTitle }}</h4>
+        <span class="sub-title--view">{{ subTitle }}</span>
+      </div>
+    </div>
+    <div class="header-right--side">
+      <fluent-icon
+        icon="arrow-up-right"
+        size="18px"
+        @click="popOutHelpCenter"
+      />
+      <fluent-icon
+        icon="arrow-swap"
+        size="28px"
+        class="portal-switch--icon"
+        @click="openSwitchPortalModal"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import Thumbnail from 'dashboard/components/widgets/Thumbnail';
+export default {
+  components: {
+    Thumbnail,
+  },
+  props: {
+    thumbnailSrc: {
+      type: String,
+      default: '',
+    },
+    headerTitle: {
+      type: String,
+      default: '',
+    },
+    subTitle: {
+      type: String,
+      default: '',
+    },
+  },
+  methods: {
+    popOutHelpCenter() {
+      this.$emit('pop-out');
+    },
+    openSwitchPortalModal() {
+      this.$emit('open');
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.sidebar-header--wrap {
+  display: flex;
+  height: 66px;
```

Comment:
Update: The design is supposed to 64px, I'll update it. Meanwhile can you make it to 64?
```suggestion
  height: var(--space-jumbo);
```